cachi2/core/package_managers/generic/main.py

```diff
@@ -79,7 +79,7 @@ def _resolve_generic_lockfile(lockfile_path: Path, output_dir: RootedPath) -> li
 
 def _load_lockfile(lockfile_path: Path, output_dir: RootedPath) -> GenericLockfileV1:
     """
-    Load the cachi2 generic lockfile from the given path.
+    Load our generic lockfile from the given path.
```

Comment:
personal preference (disregard if you want): s/our/the/

Reply:
applied